client/state/rewind/selectors/get-backup-retention-update-status.ts

```diff
@@ -0,0 +1,13 @@
+import type { AppState } from 'calypso/types';
+
```

Comment:
This selector is missing:

```
import 'calypso/state/rewind/init';
```

which is necessary to ensure that the rewind state tree will be loaded in the modular reducer.

That applies to any selector and any action creator that use a certain dynamically loaded state tree.